cmd/zed/manage/compact.go

```diff
@@ -0,0 +1,86 @@
+package manage
```

Comment:
Any reason not to put this in cmd/zed/manage/compact/command.go?

Reply:
I like it

Reply:
There's a conflict with the pre-exisiting compact package in the same directory but I want to rename that package.